mito-ai/src/Extensions/AiChat/ChatHistoryManager.tsx

```diff
@@ -1,9 +1,11 @@
 import OpenAI from "openai";
-import { Variable } from "../VariableManager/VariableInspector";
+import { IVariableManager } from "../VariableManager/VariableManagerPlugin";
+import { INotebookTracker } from '@jupyterlab/notebook';
+import { getActiveCellCode } from "../../utils/notebook";
 
 export interface IDisplayOptimizedChatHistory {
     message: OpenAI.Chat.ChatCompletionMessageParam
-    error: boolean
+    type: 'openai message' | 'connection error'
```

Comment:
I should add some documentation about this. It's not great code and should be explained.